modules/distribution-zones/src/integrationTest/java/org/apache/ignite/internal/rebalance/ItRebalanceDistributedTest.java

```diff
@@ -1087,14 +1088,18 @@ private static Set<Assignment> getPartitionStableAssignments(Node node, String t
                 .orElse(Set.of());
     }
 
-    private static Set<Assignment> getPartitionPendingAssignments(Node node, int partNum) {
-        TableViewInternal table = unwrapTableViewInternal(node.tableManager.table(TABLE_NAME));
+    private static Set<Assignment> getPartitionPendingAssignments(Node node, String tableName, int partNum) {
+        TableViewInternal table = unwrapTableViewInternal(node.tableManager.table(tableName));
 
         return Optional
                 .ofNullable(pendingPartitionAssignments(node.metaStorageManager, table, partNum).join())
```

Comment:
May I ask you to rewrite 3 places in this class where we call `.join()` on a future to something like 

```
        CompletableFuture<Set<Assignment>> fut = stablePartitionAssignments(node.metaStorageManager, table, partNum);
        
        assertThat(fut, willCompleteSuccessfully());
        
        return Optional
                .ofNullable(fut.join())
                .orElse(Set.of());

```
?

Reply:
Fixed.